src/styles/header.scss

```diff
@@ -0,0 +1,65 @@
+@import "./variables.scss";
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  &__icon {
+    margin-top: 10px;
+    width: 40px;
+    height: 40px;
```

Comment:
Just a tip, you can use mixin here to make square fo this icon. I made it also for stars :)